tests/coprocessor/test_select.rs

```diff
@@ -567,7 +567,7 @@ fn init_with_data(tbl: &ProductTable,
     store.commit();
 
     let mut end_point = Worker::new("test select worker");
-    let runner = EndPointHost::new(store.get_engine(), end_point.scheduler(), 8);
+    let runner = EndPointHost::new(store.get_engine(), end_point.scheduler(), 8, 8, 8);
```

Comment:
Try to set `end_point_txn_concurrency_on_busy` and `end_point_small_txn_tasks_limit` to a typical value, and run all the test cases with new thread pool (so that those test cases use and verify codes of the new thread pool).